controllers/upgrade_handlers.go

```diff
@@ -157,6 +160,25 @@ func (r *ImageBasedUpgradeReconciler) prePivot(ctx context.Context, ibu *lcav1al
 	}
 	ibu.ResourceVersion = rv
 
+	// Save a copy of the IBU in the current stateroot in case of uncontrolled rollback
+	ibuCopy := ibu.DeepCopy()
+	ibuCopy.ResourceVersion = ""
```

Comment:
Should we add a comment on why we reset the ResourceVersion here?

Reply:
We are doing twice same operation, lets have function for it? Like writeIBUToFile ? in which we will copy it and write without resource version ?

Reply:
Moving the ResourceVersion reset for IBU and SeedGen to the restore side, so it's in one spot, which simplifies the saving